src/ring_theory/int/basic.lean

```diff
@@ -341,3 +341,55 @@ instance decidable_int : decidable_rel (λ a b : ℤ, (multiplicity a b).dom) :=
 λ a b, decidable_of_iff _ finite_int_iff.symm
 
 end multiplicity
+
+lemma induction_on_primes {P : nat → Prop} (h₁ : P 0) (h₂ : P 1)
```

Comment:
Why not number the hypotheses to match what they state:
```suggestion
lemma induction_on_primes {P : nat → Prop} (h₀ : P 0) (h₁ : P 1)
```
You'll need to adjust the below slightly too.

Reply:
Good idea, thanks :)